.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -240,10 +240,15 @@ jobs:
           else
             TEST_COMMAND=.jenkins/pytorch/test.sh
           fi
+          PROXY_ENV=
+          if [[ $TEST_CONFIG != 'xla' ]]; then
```

Comment:
Squid proxy behaves badly for xla multiprocessing, see https://github.com/pytorch/xla/issues/3107, disabling squid proxy here for that reason

Reply:
Can you please add it as the comment to the template?